sources/Inputs/include/Behaviours.h

```diff
@@ -163,6 +163,7 @@ struct VSCConverter : public Converter {
 /// @brief Static var compensator (SVarC) behaviour
 struct StaticVarCompensator {
   using SVarCid = std::string;  ///< alias for static var compensator id
+  using BusId = std::string;  ///< alias for bus id
```

Comment:
Maybe precise if regulated bus or connection bus ?

Reply:
it is a generic alias for any king of bus, I'll modify the comment as "alias for a bus id"